config/test/smfcfg.single.test.conf

```diff
@@ -36,7 +36,6 @@ configuration:
       UPF:
         type: UPF
         node_id: 10.200.200.101
-        up_resource_ip: 192.188.2.2
```

Comment:
This is not parsed, and the gtp endpoint will be passed by the UPF on PFCP association.